### PR TITLE
Write conversation part reopened component for messages

### PR DESCRIPTION
**Pull request by daveconnis**
# What's in this PR?

Adding a 'conversation-part-reopened' component.

This PR is based off of #1638 which will need to be merged, then, once #1638 is merged, we'll need to change this base to develop.

## References: #1636, #1638

Progress on: #1597

### CODE REVIEW

**Comment by joshsmith** [comment]
You'll want to add a note in here that your other PR needs merged and this one needs its base changed before this one can be merged.

**Comment by daveconnis** [comment]
This PR is based off of #1638 which will need to be merged, then, once #1638 is merged, we'll need to change this base to develop.

**Comment by joshsmith** [comment]
@daveconnis that comment ideally would go in the description of the PR since the thread is a little buried for someone looking to decide how to merge or otherwise proceed.

File: app/components/conversations/conversation-part-reopened.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+  reopenedAt: null,
+
+  currentUser: service(),
+
+  isSelf: computed('author', 'user', function() {
+    return get(this, 'author.id') === get(this, 'user.id');
+  }),
+
+  user: alias('currentUser.user')
```
Comment (joshsmith) [inline app/components/conversations/conversation-part-reopened.js]: There are some issues here that are at odds with the style guide. See https://github.com/code-corps/code-corps-ember/blob/develop/docs/STYLEGUIDE.md#ember-modules

**Pull request merged**